src/main/java/nl/tudelft/scrumbledore/GUI.java

```diff
@@ -366,6 +372,9 @@ private void advanceLevel() {
    */
   private void checkPlayerAlive() {
     if (!game.getCurrentLevel().getPlayer().isAlive()) {
+      // Logging that the game has been restarted.
+      Logger.log("--------------------PLAYER RESTARTED THE GAME--------------------");
```

Comment:
Maybe add a function to the logger adding these "-" signs automatically in the right amount so every log line has the same length?

[feature]